content/en/tracing/livetail.md

```diff
@@ -23,49 +23,44 @@ further_reading:
+The APM [Live Search][1] gives you the ability to search all ingested spans using any tag in real-time for the past 15 minutes. It displays spans as soon as they are sent by the Datadog agent and before they are indexed by Datadog. All spans ingested by Datadog are displayed without any sampling (Tracing without Limits\*). With the APM Live Search you can:
 
-- Write search parameters to refine the stream of traces.
-- View distributed trace in real-time.
-- Add or remove columns from span tags for a customized view.
+- Write search queries to refine the stream of traces by any tag on any span. Example: monitor that a new deployment went smoothly by filtering on version_id of all tags
+- View 100% of trace spans as they are ingested - no sampling on ingested spans for 15 minutes. Example: view outage related information in real-time as it happens related to specific org_id or customer_id associated with a child span
+- Search queries autocomplete in real-time. Example: check if a process has correctly started type process_id of a child span tag and it auto completes the id.
+- View Live timeseries visualization of key RED metrics: requests per second, errors, and latency. Example: monitor load test and performance impact on your endpoints by filtering on the duration of a child resource
+- One-click search queries on any span or tag directly from the trace panel view
+- Add, remove, and sort columns from span tags for a customized view
 
-This feature allows you, for instance, to check if a process has correctly started, or if a new deployment went smoothly. Or view outage related information in real-time
+## Live Search mode
+{{< img src="tracing/live_tail/livesearch_mode.gif" alt="Live Search mode" >}}
 
-## Live Tail view
-
-Choose the `Live Tail` option in the time range selector to switch to the Live Tail view. The number of received spans per second is displayed at the top left, as well as the sampling rate. Since a stream of thousands of spans per second is not human readable, high throughput span streams are sampled for visual clarity. Use the Live Tail search bar filtering features to filter the spans stream and the Pause/Play button at the top right of the screen to pause or resume the stream.
+`Live Search` mode is the default experience on the Traces page or you can choose the `LIVE` option in the time range selector to switch to the Live Search mode for the past 15 minutes from the `HISTORICAL` mode. The number of received spans per second is displayed at the top of the traces table, as well as the sampling rate. Since a stream of thousands of spans per second is not human readable, high throughput span streams are sampled for visual clarity but are still searchable. Use the Live Search query bar filtering features to filter the spans stream and the Pause/Play button at the top right of the screen to pause or resume the stream.
```

Comment:
```suggestion
`Live Search` mode is the default experience on the Traces page or you can choose the `LIVE` option in the time range selector to switch to the Live Search mode for the past 15 minutes from the `HISTORICAL` mode. The number of received spans per second is displayed at the top of the traces table, as well as the sampling rate. Since a stream of thousands of spans per second is not human readable, high throughput span streams are sampled for visual clarity but are still searchable. Use the Live Search query bar filtering features to filter the spans stream and the **Pause/Play** button at the top right of the screen to pause or resume the stream.
```